app/models/piece.rb

```diff
@@ -1,8 +1,21 @@
 class Piece < ActiveRecord::Base
+  belongs_to :user
   belongs_to :game
 
   def is_obstructed?
     return false
   end
 
+#Capture Pieces Methods - Validates if the other piece is valid to capture, moves current piece to new location
+
+  def move_to!(new_x, new_y)
+    capture_piece = Piece.find_by(x_cord: new_x, y_cord: new_y)
+    return if capture_piece && capture_piece.color == self.color
```

Comment:
well done!

one more thing I'd like to see is to abstract out the checks of the color.. suggestion:

return if capture_piece && own_piece?(capture_piece)

def own_piece? (piece)
# to implement
end

This way you don't care about the color anymore, but about the core logic which is "My own piece or my opponents's piece"